platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -115,16 +115,52 @@ export default class DisplaySetService extends PubSubService {
     return this.activeDisplaySets;
   }
 
+  /**
+   * Gets the set of series with this series instance UID
```

Comment:
wrong documentation, get displaysets that include series instanceuid (i think ...)